aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,106 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get Started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you will create a SignalR on ASP.NET Core application.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core.
+
+  ![Solution](get-started-signalr-core/_static/signalr-get-started-finished.png)
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Create a SignalR on ASP.NET Core web app.
+> * Create a SignalR hub to push content to clients.
+> * Modify the `Startup` class to configure the app.
+> * Use the SignalR JavaScript library to send messages and display updates from the hub.
+
+# Prerequisites
+
+Install the following:
+
+* [.NET Core 2.1.0 Preview 1 SDK](https://www.microsoft.com/net/core) or later.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
+
+## Create an ASP.NET Core project that uses SignalR
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
+
+1. Select **Web Application** which creates a project using Razor Pages. Then select **Ok**. Be sure that **ASP.NET Core 2.1** is selected from the framework selector.
```

Comment:
@anurse  can we add something like. The Razor Pages app is used to create a basic client app and host the server. I'd like to tell they why we're using RP (because it's easy) but give them an idea of what they could do with other clients.

Reply:
Need to tell them to name it SignalRChat so the ns matches.